webapp/portlet/src/main/resources/locale/portlet/Portlets_en.properties

```diff
@@ -316,6 +316,10 @@ SpaceSettings.button.moveAfter=Move after
 SpaceSettings.title.addApplication=Space application installer
 SpaceSettings.title.deleteApplicationConfirm=Delete application ?
 SpaceSettings.deleApplicationConfirm=Would you like to remove application {0} from space? (application data will be kept)
+SpaceSettings.application.members.title=Members
```

Comment:
seems unrelated

Reply:
this was added to the PR in order to resolve a conflict